CMakeLists.txt

```diff
@@ -12,7 +12,15 @@ project(bpfilter
 include(GNUInstallDirs)
 
 find_package(Doxygen REQUIRED)
-find_package(cmocka REQUIRED)
+
+find_package(PkgConfig)
+pkg_check_modules(bpf REQUIRED IMPORTED_TARGET libbpf)
+pkg_check_modules(elf REQUIRED IMPORTED_TARGET libelf)
+pkg_check_modules(cmocka REQUIRED IMPORTED_TARGET cmocka)
```

Comment:
```
find_package(cmocka REQUIRED) # We can keep this, and link `tests_unit` to `cmocka::cmocka`
find_package(PkgConfig REQUIRED)

pkg_check_modules(bpf REQUIRED IMPORTED_TARGET libbpf)
pkg_check_modules(elf REQUIRED IMPORTED_TARGET libelf)
```

Reply:
@qdeslandes The `cmocka` CMake package script does not set up a target so we would have to use `${CMOCKA_LIBRARIES}` everywhere. I prefer using pkgconfig which does give us a target, but it's your call.

Reply:
After checking on Ubuntu, it doesn't. Ubuntu has `libcmocka-dev-1.1.5` vs `cmocka-devel-1.1.7` on Fedora. Let's go with your way then!